keyboards/ymdk/ymd67/keymaps/vial/rules.mk

```
@@ -0,0 +1,11 @@
+VIA_ENABLE = yes
+VIAL_ENABLE = yes
+RGBLIGHT_ENABLE = no
+BACKLIGHT_ENABLE = no
+BLUETOOTH_ENABLE = no
+AUDIO_ENABLE = no
+MIDI_ENABLE = no
+MOUSEKEY_ENABLE = no
+POINTING_DEVICE_ENABLE = no
+TERMINAL_ENABLE = no
+EXTRAFLAGS += -flto
```

Most of these are disabled by default and that particular set of extraflags has been canonicallized into `LTO_ENABLE`.
```suggestion
MOUSEKEY_ENABLE = no
LTO_ENABLE = yes
```